components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/commons/js/dialog/error-handler-dialog.js

```diff
@@ -731,8 +731,73 @@ define(['require', 'lodash', 'jquery', 'constants', 'backbone', 'alerts', 'pagin
+                                                             wrappedErrorEntry.modifiablePayloadString + '</textarea>');
+                    } else if (wrappedErrorEntry.errorEntry.eventType === 'REPLAYABLE_TABLE_RECORD') {
+                        var modifiablePayloadJson = JSON.parse(wrappedErrorEntry.modifiablePayloadString);
+                        var editableTable = $('<table></table>');
+                        var tableHeader = $('<tr></tr>');
+                        modifiablePayloadJson.attributes.forEach(function (attribute) {
+                            tableHeader.append('<th><span class="event-payload-table-header-title">' + attribute.name
+                            + '</span> <span class="event-payload-table-header-type">(' + attribute.type
+                            + ')</span></th>');
+                        ;});
+                        editableTable.append(tableHeader);
+                        modifiablePayloadJson.records.forEach(function (record) {
+                            var tableRow = $('<tr></tr>');
+                            record.forEach(function (column) {
+                                tableRow.append('<td><input type="text" class="event-payload-table-input" value="' + column +
+                                '"></td>');
+                            });
+                            editableTable.append(tableRow);
+                        ;});
+                        return editableTable;
+                    }
+                },
+
                 renderOriginalPayload: function(errorEntry) {
-                    var originalPayload = $('<div><h4>Original Payload</h4></div>');
+                    var originalPayload = $('<div></div>');
+                    if (errorEntry.eventType === "REPLAYABLE_TABLE_RECORD") {
+                        var unmodifiablePayloadJson = JSON.parse(errorEntry.originalPayload);
+                        var uneditableTable = $('<table></table>');
+                        var tableHeader = $('<tr></tr>');
+                        unmodifiablePayloadJson.attributes.forEach(function (attribute) {
+                            tableHeader.append('<th><span class="event-payload-table-header-title">' + attribute.name
+                                + '</span> <span class="event-payload-table-header-type">(' + attribute.type
+                                + ')</span></th>');
+                        ;});
+                        uneditableTable.append(tableHeader);
+                        unmodifiablePayloadJson.records.forEach(function (record) {
+                            var tableRow = $('<tr></tr>');
+                            record.forEach(function (column) {
+                                tableRow.append('<td><input type="text" class="event-payload-table-input" value="'
+                                + column + '" disabled></td>');
+                            });
+                            uneditableTable.append(tableRow);
+                        ;});
```

Comment:
There's an extra `;` in the front it seems. Let's remove that. There are some other similar cases as well.

Reply:
Fixed in bbaeabc